AnkiDroid/src/main/java/com/ichi2/anki/ModelFieldEditor.java

```diff
@@ -429,6 +435,7 @@ private void sortByField() {
             mCol.modSchema();
             TaskManager.launchCollectionTask(new CollectionTask.ChangeSortField(mMod, mCurrentPos), listener);
         } catch (ConfirmModSchemaException e) {
+            Timber.w(e);
```

Comment:
Okay - I'll push back on the log statement and ask for it (but happy to discuss), this should have been a `Timber.v`. 
```suggestion
            Timber.v(e);
```

Reply:
Timbers are changed to v.
Note that this does not seems to be consistent with existing code, which deals with exceptions in various ways. Sometime using printStackTrace, sometime timbering with various degree of severity.

I probably won't try to uniformize them, I don't see the point and looking at all catch took a lot of time and can't really be automatized unless I know how to look at java's AST